projects/packages/videopress/src/client/block-editor/blocks/video/components/player/index.native.js

```diff
@@ -171,15 +182,7 @@ export default function Player( { isSelected, attributes } ) {
 		<View style={ [ style[ 'videopress-player' ], { aspectRatio } ] }>
 			{ overlay }
```

Comment:
Nitpick: If the overlay is only used to prevent touch events get to the player. There's an alternative using the [`pointerEvents` prop](https://reactnative.dev/docs/view#pointerevents).

```
<View
	style={ [ style[ 'videopress-player' ], { aspectRatio } ] }
	pointerEvents={ isSelected ? 'auto' : 'none' }
>
	{ showLoadingOverlay && loadingOverlay }
	{ html && sandbox }
</View>
```

Reply:
fixed in 2d886e2aa3e954fc02ff9a00d35458abe9d424a6